content/en/opentelemetry/integrations/runtime_metrics/_index.md

```diff
@@ -9,36 +9,202 @@ type: multi-code-lang
+Datadog supports OpenTelemetry runtime metrics for the following languages:
 - Java
 - .NET
 - Go
 
-## Metric naming conventions
+For details about host and container metrics mapping, read [OpenTelemetry Metrics Mapping][1].
 
-Runtime metrics follow different naming conventions depending on their source: OpenTelemetry Collector Datadog Exporter, Datadog Agent OTLP Ingestion, or Datadog tracing libraries. When using OpenTelemetry runtime metrics with Datadog, you receive both the original OpenTelemetry runtime metrics as well as mapped Datadog runtime metrics for equivalent metrics. Runtime metrics have the following prefixes which indicate their source:
+## Setup instructions
 
-| OTel Collector Datadog Exporter | Datadog Agent OTLP Ingest |  Datadog tracing library |
-| --- | --- | --- |
-| `otel.process.runtime.*` | `process.runtime.*` | `runtime.<LANG>.*` |
+### 1. Prerequisites
 
-**Note**: OpenTelemetry runtime metrics are mapped to Datadog by metric name. Don't do mapping renaming of host metrics for OpenTelemetry runtime metrics or it will break.
+- You are successfully [sending OpenTelemetry metrics to Datadog][2].
+- You have installed the corresponding language integration.
 
-For details about host and container metrics mapping, read [OpenTelemetry Metrics Mapping][1].
+### 2. Configure your application
+
+Select your language to see instructions for configuring the OpenTelemetry SDK to send runtime metrics:
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+If you have instrumented your Java applications with [OpenTelemetry automatic instrumentation][3], runtime metrics are automatically enabled.
+
+If you have instrumented your Java application with [OpenTelemetry manual instrumentation][4], see the following guides for your Java version:
+- [Java 8][5]
+- [Java 17][6]
+
+[3]: https://opentelemetry.io/docs/instrumentation/java/automatic/
+[4]: https://opentelemetry.io/docs/instrumentation/java/manual/
+[5]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-telemetry/runtime-telemetry-java8/library
+[6]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-telemetry/runtime-telemetry-java17/library
+
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+OpenTelemetry (OTel) Go applications are [instrumented manually][3]. To enable runtime metrics, see the documentation for the [runtime package][4].
+
+
+[3]: https://opentelemetry.io/docs/instrumentation/go/manual/
+[4]: https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/runtime
+
+{{% /tab %}}
+
+{{% tab ".NET" %}}
+
+<div class="alert alert-warning">The minimum supported version of the .NET OpenTelemetry SDK is <a href="https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.5.0">1.5.0</a></div>
+
+If you have instrumented your .NET applications with [OpenTelemetry automatic instrumentation][3], runtime metrics are automatically enabled. 
```

Comment:
This is probably out of the scope of this PR, but the .NET section likely needs to be updated since the runtime metric names have changed starting .NET 9: https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-runtime.